### DISCUSSION TITLE
明明摆烂是他们最好的选择，但为什么却执行不起来？

### DISCUSSION THREAD

**Discussion by Dreaming-Pepper** [WeNeedHome/SummaryOfLoanSuspension#1155] [General]
### 打英雄联盟有感

今天打了把极地大乱斗，石头空了一个大，队友开喷，我直接一句“等下我就摆烂”🤙，队友态度180°大转弯，直接喊“石头哥，冲”，顺势拿下比赛。显而易见，做工具人，没法C，只能好好当工具人。

这群买期房的，明明就是韭菜，以为自己能混点房地产的汤油花。当然不是笑他们，风险和收益是对等的，第一个吃“河蟹生腌”，多少要给点鼓励👆。

合同里写着清清楚楚，我读书时就知道，这里面这么恶心，居然还去买，什么心态😅。现在我出来上班了，他们烂尾了🤣。

摆在他们面前最好的结局。
一：房子建起来😅（怎么可能？能建早建了）；
二：政府和银行收烂摊子，韭菜们保住了部分茎叶🤣（想得美，房贷只占银行营业很小一部分，直接拖死他们）；

唯一可行的解就是【 **摆 烂** 】，超级大败烂，24小时不间断摆给全国人民看，让全世界人一起看这些人对他们的人民做了什么，要主动构陷。找漂亮国说不定还能把钱挣回来。打压受阻？曝光啊，像微博那些女权曝光写小作文一样啊！胜利之前一切都是手段，之前上海疫情想惩罚子孙三代，直接回怼“这是最后一代！”，我想说，希望至始至终🤣。

显然，就目前的讨论热度，部分发起者也被请去喝茶，本想坐等看烟火，但期望估计是落空，大家都在怕，说白了就是思想钢印太深了。

顾前顾后，解决方法很简单：

- 如果是利益受损，请去斗争。
- 如果被剥夺的一无所有，请用生命斗争。

毕竟你不是国家的人民，国家不会以保护真正的人民的方式，保护屁民。要不是国人素质太高，不像欧美人那么“难管理”。乐于奉献自己为集体，为宏大叙事规训自己🤣。

我对祖国的爱，正如祖国爱我般❤。
不要说我没同理心，我不是韭菜，不可能和韭菜共情。
但声音能传递下去，能让死灰复燃，我愿近微薄之力。

_如果我删了此Discussions，我想说说明应该清楚了_

**Comment by Cegazhos** [comment]
热搜还不是狗官能控制的 只有停贷才是正解。

**Reply by Dreaming-Pepper** [reply]
仅仅停个贷是远远不够的，要停贷给全国人看，二十小时不间断的刷，要出点钱去刷舆论，出钱写小作文。今天刷到[抗议电价上涨80%，苏格兰民众在能源监管大楼前焚烧电费账单](https://www.bilibili.com/video/BV16t4y1E7zW)。
瞧瞧人家，只不过人家是全苏格兰，这边仅只有烂尾楼住户，一边是过分的电费，另一边很可能直接压垮一个男人，拆散一对夫妻，重击一个家庭孩子住烂尾楼了。
笑了，说不定小孩子上课，班里人估计都笑他没个像样的房子，活着真累。

**Reply by Cegazhos** [reply]
> 仅仅停个贷是远远不够的，要停贷给全国人看，二十小时不间断的刷，要出点钱去刷舆论，出钱写小作文。今天刷到[抗议电价上涨80%，苏格兰民众在能源监管大楼前焚烧电费账单](https://www.bilibili.com/video/BV16t4y1E7zW)。 瞧瞧人家，只不过人家是全苏格兰，这边仅只有烂尾楼住户，一边是过分的电费，另一边很可能直接压垮一个男人，拆散一对夫妻，重击一个家庭孩子住烂尾楼了。 笑了，说不定小孩子上课，班里人估计都笑他没个像样的房子，活着真累。

狗官能控制热搜😓我不跟家里人说 他们根本不可能知道710白衣人。有时候 互联网不一定是好事。
现在这样就挺好的，等财政兜底，或者住户 银行一起吃亏。

**Comment by GithubRobot01** [comment]
你这就像对出车祸的人说你为什么坐车一样,有时候买不买房也不完全取决你

**Reply by Dreaming-Pepper** [reply]
非也，讨论重心不在 大骂为什么明知车祸率很高，依然坐车。

而是，出了车祸，应该找责任人要回应得的权益，而非起身拍拍屁股灰就走觉得没什么大不了的。

我主要目的是驱赶他们行动，去反抗如此恶劣的剥削。

为什么我不行动？

据我所知，如果不是一条路上的人，根本不会在意他人的苦难，也就是，如果我是烂尾楼业主，我恰好已经没啥退路（例如，家里种没几亩地的田，一贫如瘠，父母年过半百，90年初独生子女的我是小镇做题家，刚贷完百万房贷和天价彩礼，遇新冠爆发经济下行中被裁员的996程序员，年薪就17w吧）结果房子直接给你烂尾，不仅钱没了，还一屁股债。玩毛？这不冲塔？结果心里一句，冲塔没用的，冰冷铁拳的制裁还是会打向自己。

我一想到这就浑身、难受、恶心，要不是国人太好管理，这帮人有这机会？

此外继续用这个车祸率做比喻就变成言语陷阱，偷换概念了，根本不能相比；

**Comment by a6749920** [comment]
政府不想降房价，影响一部分不想，都不愿意当这个冤大头，二手房自由买卖都限制，房价蹦了，可想而知经济要蹦了
